src/app/app.css

```diff
@@ -37,6 +37,18 @@ SOFTWARE.
 */
 html, body, #root {
   height: 100%;
+  overflow: hidden;
```

Comment:
Any reason to set overflow as hidden here?

Reply:
Same for `.pf-c-page__sidebar.pf-m-collapsed`.

Reply:
The reason why the overflow is hidden is because if you open the tour you are able to start scrolling past the content a little. I could not really pinpoint why other than it may be a problem with `react-joyride`, but putting this attribute solves the problem.

Reply:
![image](https://user-images.githubusercontent.com/69827283/228148054-84d3c770-d429-4f86-814d-755548b579f5.png)

As for the side bar being collapse, this is what happens if the css rule is turned off, but once again, I'm not really sure why. I think it's because `react-joyride` sets styles on the highlighted elements to some overflow attribute. I can actually see it here, inlined: 
![image](https://user-images.githubusercontent.com/69827283/228148634-6ab70b2c-ab75-4f01-8b03-3f75f3285266.png)

Reply:
> Any reason to set overflow as hidden here?

I realize that whitespace is because of this issue: https://github.com/cryostatio/cryostat-web/issues/863

Reply:
> I realize that whitespace is because of this issue

Yehhh just the skip-to-content is out of place. So we should not set hidden here as overflow content on body should be scrollable or else things appear stuck.

Reply:
> As for the side bar being collapse, this is what happens if the css rule is turned off. I think it's because react-joyride sets styles on the highlighted elements to some overflow attribute. I can actually see it here, inlined:

Ahh i see thanks!